Small GUI tweaks, added logo & simple file menu

Narwher opened the pull request:
This is a very useful application!! I've added a logo (just a placeholder for now) and made some minor changes to the user interface.

Also added a requirements.txt to make installation of dependencies a bit easier, as well as a sample docx and csv.

Try it out before merging and see if you like the changes!

Narwher committed:
Include i/o examples

Narwher committed:
add logo™

Narwher committed:
Create requirements.txt

for simpler client installation

Narwher committed:
GUI tweaks, etc

- remove custom msg box in favor of tkinter built in dialogs

+ simple file menu (needs work)
+ condense window size, buttons
+ add text field for file browser
+ fixed small 'previous button' bug in convert menu

danieleremchuk commented on app.py:
```diff
@@ -268,17 +258,17 @@ def start_convert(self):
-    w = 600 
-    h = 600 
+    w = 400
+    h = 300
```
Need to increase the height to at least 350. Otherwise, progress bar is hidden in the ConvertFile frame.

Narwher replied:
oops.jpg 
Will revise!

danieleremchuk commented on requirements.txt:
```diff
@@ -0,0 +1,5 @@
+ConfigObj
+files
```
Do we need to add this to requirements? "files" refers to my files.py. I can rename that file to help clarify that (to "read_file.py" or something like that?).

Narwher replied:
'files' is a library that can be installed, so a rename of the module might be in order.

edit: goes for 'validate' as well. since custom error message boxes are removed, there are only two functions left in 'validate.py'. They could probably be merged with another file.

danieleremchuk commented on requirements.txt:
```diff
@@ -0,0 +1,5 @@
+ConfigObj
+files
+pillow
+pymsgbox
```
If we opt to use the tkinter standard messagebox, we may not need to include pymsgbox as this library is no longer used.

danieleremchuk commented on app.py:
```diff
@@ -24,6 +20,24 @@ def __init__(self, *args, **kwargs):
         window.grid_rowconfigure(0, weight=1)
         window.grid_columnconfigure(0, weight=1)
 
+        # menu bar (wip) - maybe link functions here as well? -jh
```
We can definitely consider tweaking these things to do things later on, but I like this item as a work-in-progress concept.

danieleremchuk commented on app.py:
```diff
@@ -222,15 +212,15 @@ def __init__(self, parent, controller):
         progress_text = tk.Label(midFrame, textvariable=progress_txt)
         button_main = tk.Button(midFrame, text="Main Menu", command=lambda: controller.show_frame("MainPage"), padx=10, pady=5)
-        button_prev = tk.Button(midFrame, text="Previous", command=lambda: controller.show_frame("FindFile"), padx=10, pady=5)
+        button_prev = tk.Button(midFrame, text="Previous", command=lambda: controller.show_frame("BreweryName"), padx=10, pady=5)
```
Good catch!

danieleremchuk reviewed (commented):
The changes are all great, but there are a few items that I'd like you to review before I push through the changes. Let me know if you have any questions.

Narwher committed:
Removed sample dox

Narwher committed:
Merged validate.py → file.py

+changed 'files.py' to 'file.py' to distinguish module

-Removed pymsgbox lib and warning messages from validate.py

+Renamed module/function references in app.py accordingly

Narwher committed:
Remove unnecessary requirements

Originally confused 'validate' and 'files' with libraries as opposed to the local modules. Whoops!

pymsgbox removed in favor of tkinter.messagebox (may use for custom dialogs later?)

Narwher committed:
Update app.py

+Rename refrences to newly named modules

+Change height from 300px → 350px to properly display progress bar

Narwher commented:
Revisions pushed! Biggest change was the merging of functions from validate.py into file.py (formerly files.py). Doing this prevents any confusion with library names. Also, removed unnecessary requirements from the text file.

danieleremchuk reviewed (approved):
I like this change. It makes sense to have all file management and validation in one file, aptly named file.py. And good adding the comment to the file to state that. Approved on my end!

danieleremchuk reviewed (approved):
Looks good - still gets the job done but keeps things concise. Approved on my end!

danieleremchuk reviewed (approved):
All changes look good and consistent. Approved on my end!
